CMake/functions.cmake

```diff
@@ -324,6 +327,9 @@ function(install_library_targets)
         if(${_type} STREQUAL SHARED_LIBRARY)
           export(TARGETS ${_target} APPEND FILE "${VisGUI_EXPORT_FILE}")
         endif()
+      elseif(${_type} STREQUAL "INTERFACE_LIBRARY")
```

Comment:
This should be `_type`, not `${_type}`. Putting `"INTERFACE_LIBRARY"` in quotes is correct, and the others above should do the same thing. Please fix.

Reply:
Good catch, thanks.